<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QbsProjectManager::Internal::QbsBuildStepConfigWidget</class>
 <widget class="QWidget" name="QbsProjectManager::Internal::QbsBuildStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>121</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="buildVariantLabel">
     <property name="text">
      <string>Build variant:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="buildVariantComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="jobSpinBox"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="jobsLabel">
     <property name="text">
      <string>jobs</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="qmlDebuggingLabel">
     <property name="text">
      <string>Enable QML debugging:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="qmlDebuggingLibraryCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qmlDebuggingWarningIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/projectexplorer/images/compile_warning.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qmlDebuggingWarningText">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="propertyLabel">
     <property name="text">
      <string>Properties:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QbsPropertyLineEdit" name="propertyEdit"/>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="dryRunCheckBox">
       <property name="text">
        <string>Dry run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="keepGoingCheckBox">
       <property name="text">
        <string>Keep going</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="checkBoxSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QbsPropertyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qbspropertylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
